### PR TITLE
1 test driver

### PR DESCRIPTION
**Pull request by TakeshiYoshikawa**

### CODE REVIEW

File: mobile/test_driver/app_test.dart [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      await driver.tap(find.byValueKey("createStudentButton"));
+      await driver.tap(find.byValueKey("studentFirstName"));
+      await driver.enterText("Ronaldo");
+      await driver.tap(find.byValueKey("studentLastName"));
+      await driver.enterText("Azevedo");
+      await driver.tap(find.byValueKey("addStudent"));
+      expect(await driver.getText(find.byValueKey("1_fullName")), "Ronaldo Azevedo");
+    });
+  });
+}
```
Comment (VitorCantao) [inline mobile/test_driver/app_test.dart:90]: I think dividing each feature test into it's own file will help keep things organized.

For example: you want to test things related to classroom, so it has to be on the `classroom_test`, or something like it.

File: mobile/test_driver/app_test.dart [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      driver = await FlutterDriver.connect();
+    });
+
+    tearDownAll(() async {
+      if (driver != null) {
+        driver.close();
+      }
+    });
+
+    test("Login an account properly validated", () async {
```
Comment (VitorCantao) [inline mobile/test_driver/app_test.dart:18]: Where are the edge cases? We need tests to make sure things are properly working

**Review by VitorCantao — CHANGES_REQUESTED** [review]
